### PR TITLE
git auto-commit action

### PR DESCRIPTION
**Pull request by queilawithaQ**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const exec = (cmd, args=[]) => new Promise((resolve, reject) => {
 });
 
 const main = async () => {
-    await exec('bash', [path.join(__dirname, './entrypoint.sh')]);
+    await exec('sh', [path.join(__dirname, './entrypoint.sh')]);
```
Comment (queilawithaQ) [inline index.js:27]: exchange 'bash' ----> 'sh'

File: entrypoint.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (queilawithaQ) [inline entrypoint.sh:1]: 'bash' ===> 'sh'

**Review by queilawithaQ — COMMENTED** [review]
### Entrypoint.sh 
``set -eu`` on line 3

**Comment by stefanzweifel** [comment]
@queilawithaQ Hi, could you please note use this repository as your testing ground? Or at least provide a meaningfull title and description of the pull request

**Pull request closed without merging**